File: CS50's_Introduction_to_CS/Week2/scores.c
//
// Created by saksham on 20-06-2021.
//

#include <stdio.h>
#include <cs50.h>

const int TOTAL = 4;

float average(int length, int array[]);

int main(void)
{
    // Declaring scores array using constant
    int scores[TOTAL];
    for (int i = 0; i < TOTAL; i++)
    {
        // Taking 4 scores input
        scores[i] = get_int("Score: ");
    }

    // Printing and calling average function.
    printf("Average: %f\n", average(TOTAL, scores));
}

float average(int length, int array[])
{
    int sum = 0;
    // Adding numbers from the array
    for (int i = 0; i < length; i++)
    {
        sum = sum + array[i];
    }
    // Returning value of average.
    return sum / (float)length;
}

File: CS50's_Introduction_to_CS/Week6/Week2/scores.py
# Averages three numbers using a list
# from cs50 import get_int

# Scores
scores = [72, 73, 33]
# Print average
print("Average: " + str(sum(scores) / len(scores)))
print(f"Average: {sum(scores) / len(scores)}")

# Getting user's input:
scores = []
for i in range(3):
    # scores.append(get_int("Score: "))
    scores.append(int(input("Score: ")))
# Print average
print(f"Average: {sum(scores) / len(scores)}")
